.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,56 @@
-**What type of PR is this?**
-> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
->
-> /kind api-change
-> /kind bug
-> /kind cleanup
-> /kind design
-> /kind documentation
-> /kind failing-test
-> /kind feature
-> /kind flake
-
-**What this PR does / why we need it**:
-
-**Which issue(s) this PR fixes**:
+<!-- Thanks for sending a pull request! Here are some tips for you:
+
+1. If you want **faster** PR reviews, read how: https://github.com/kubesphere/community/blob/master/developer-guide/development/the-pr-author-guide-to-getting-through-code-review.md
+2. In case you want to know how your PR got reviewed, read: https://github.com/kubesphere/community/blob/master/developer-guide/development/code-review-guide.md
+3. Here are some coding convetions followed by KubeSphere community: https://github.com/kubesphere/community/blob/master/developer-guide/development/coding-conventions.md
+
+-->
+
```

Comment:
I think there are two redundant blank lines for removal.

Reply:
There are all comments, and not shown in the PR description, adding blank lines to make it more readable.